doc/sr_plugins.dox

```diff
@@ -21,7 +23,7 @@ on large data. In this case the whole data must be parsed, the change performed,
 back. Instead, another datastore may be able to store the change directly without parsing and printing all the other
 stored data. As for the space required, if implementing a datastore for a specific YANG module, the data structure can
 fully depend on the schema nodes defined in the module and hence avoid storing any redundant information. Finally,
-note that the internal LYB datastore is also implemented as a datastore plugin so it can be used as an example
+note that the internal JSON datastore is also implemented as a datastore plugin so it can be used as an example
```

Comment:
This should be updated but there were 3 mentions of `LYB` in this text, not just one.